pom.xml

```diff
@@ -1,96 +1,92 @@
-    <version>3.8.0-02</version>
+    <version>3.19.0-SNAPSHOT</version>
   </parent>
-  <groupId>org.sonatype.nexus.plugins</groupId>
-  <artifactId>nexus-repository-cpan</artifactId>
-  <name>${project.groupId}:${project.artifactId}</name>
-  <version>1.0.0</version>
-  <inceptionYear>2018</inceptionYear>
-  <packaging>bundle</packaging>
-
-  <dependencies>
-    <dependency>
-      <groupId>org.sonatype.nexus</groupId>
-      <artifactId>nexus-plugin-api</artifactId>
-      <scope>provided</scope>
-    </dependency>
-
-    <dependency>
-      <groupId>org.sonatype.nexus</groupId>
-      <artifactId>nexus-repository</artifactId>
-      <scope>provided</scope>
-    </dependency>
-
-    <dependency>
-      <groupId>org.sonatype.nexus</groupId>
-      <artifactId>nexus-rapture</artifactId>
-      <scope>provided</scope>
-    </dependency>
-
-    <dependency>
-      <groupId>org.sonatype.goodies</groupId>
-      <artifactId>goodies-testsupport</artifactId>
-      <scope>test</scope>
-    </dependency>
-
-    <dependency>
-      <groupId>org.apache.commons</groupId>
-      <artifactId>commons-compress</artifactId>
-    </dependency>
-
-    <dependency>
-      <groupId>org.yaml</groupId>
-      <artifactId>snakeyaml</artifactId>
-    </dependency>
-
-  </dependencies>
-
-  <build>
-    <plugins>
-      <plugin>
-        <groupId>org.sonatype.nexus.buildsupport</groupId>
-        <artifactId>extjs-maven-plugin</artifactId>
-        <configuration>
-          <namespace>NX.cpanui</namespace>
-        </configuration>
-      </plugin>
-
-      <plugin>
-        <groupId>org.sonatype.plugins</groupId>
-        <artifactId>yuicompressor-maven-plugin</artifactId>
-      </plugin>
-
-      <plugin>
-        <groupId>org.apache.karaf.tooling</groupId>
-        <artifactId>karaf-maven-plugin</artifactId>
-      </plugin>
-
-      <plugin>
-        <groupId>org.apache.maven.plugins</groupId>
-        <artifactId>maven-surefire-plugin</artifactId>
-        <version>2.20</version>
-      </plugin>
-
-    </plugins>
-  </build>
-</project>
\ No newline at end of file
+  <artifactId>cpan-parent</artifactId>
+  <version>0.0.0</version>
+  <packaging>pom</packaging>
+  <name>${project.groupId}:${project.artifactId}-parent</name>
+  <inceptionYear>2019</inceptionYear>
+  <modules>
+    <module>nexus-repository-cpan</module>
+    <module>nexus-repository-cpan-it</module>
+  </modules>
+  <distributionManagement>
+    <snapshotRepository>
+      <id>ossrh</id>
+      <url>https://oss.sonatype.org/content/repositories/snapshots</url>
+    </snapshotRepository>
+  </distributionManagement>
+  <properties>
+    <nxrm-version>3.19.0-SNAPSHOT</nxrm-version>
+  </properties>
+  <repositories>
+    <!-- ensure we can find the parent pom when starting from an empty local .m2 repository -->
+    <repository>
+      <id>rso-snapshots</id>
+      <url>https://repository.sonatype.org/content/repositories/snapshots</url>
+    </repository>
+  </repositories>
```

Comment:
Optional enhancement (already included in [nexus-format-archetype](https://github.com/sonatype-nexus-community/nexus-format-archetype)): Detect and require java 8. Could do so by adding the snippet below:

```
<build>
  <plugins>
      <!--
      Ensure use of Java 8, as required by NXRM
      -->
      <plugin>
        <artifactId>maven-enforcer-plugin</artifactId>
        <version>1.4.1</version>
        <executions>
          <execution>
            <id>enforce-java</id>
            <goals>
              <goal>enforce</goal>
            </goals>
            <configuration>
              <rules>
                <requireJavaVersion>
                  <version>[1.8,1.9)</version>
                </requireJavaVersion>
              </rules>
            </configuration>
          </execution>
        </executions>
      </plugin>
  </plugins>
</build>
```

Reply:
In: a277059